LOG.tbasicu

```diff
@@ -150,33 +155,32 @@ Type tLogThis
     Local sTagF as String
     Local sDT as String
     Local sDTF as String
-    Local successFile as Number
-    Local logWhatTitleFile = logWhatTitle
+    Local logWhatTitleFile as String = logWhatTitle
     
-    '[t] Workaround for Switch$ error ( see https://www.thinbasic.com/community/project.php?issueid=616 ) ...
-    local TagDebug = me.TagDebug 
```

Comment:
Beware! Doing `local variable = something`, thinBasic will create a variable of VARIANT data type! That would not result in 1:1 conversion.

Reply:
More, this is totally illogic : we declare a local _variable_ to make possible the use of _me.variable_ ! But this workaround... works! ([bug1 ](https://www.thinbasic.com/community/project.php?issueid=616)& [bug2 ](https://www.thinbasic.com/community/project.php?issueid=617) reported) (ok, bug2 is a bit particular)

Reply:
Do you mind leaving the '{t] tag at the start of the line for it is shown in thinAir's treeview ?

Reply:
I am sure the workaround works in this case, but it could not in some more complex cases, that is why I pointed it out.

I fixed the `' [t]`, please, just refresh.

Reply:
refreshed, but the [t] still on the #region line. Did you push to github ?

Reply:
It should be okay now, please have a look :)